build/lib/release.sh

```diff
@@ -173,14 +174,24 @@ function kube::release::package_node_tarballs() {
 # Package up all of the server binaries in docker images
 function kube::release::build_server_images() {
   kube::util::ensure-docker-buildx
+  docker buildx create --name img-builder --use || true
+  docker buildx inspect --bootstrap
+  kube::util::trap_add 'docker buildx rm' EXIT
+
+  platforms=("${KUBE_SERVER_PLATFORMS[@]}")
+  if [[ "${KUBE_BUILD_WINDOWS}" =~ [yY] ]]; then
+    platforms=("${KUBE_SERVER_PLATFORMS[@]}" "windows/amd64")
```

Comment:
this override is *really* hard to follow ... it makes KUBE_SERVER_PLATFORMS non-authoritative for `kube::release::build_server_images`, and relies on a bash variable not defined in this file ... 

why override this way instead of invoking with KUBE_SERVER_PLATFORMS=("windows/amd64") or something similar?